juno/assetlist.json

```diff
@@ -1218,7 +1218,28 @@
       "symbol": "CATMOS",
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/catmos.png"
-      }      
+      },
+      {
+      "description": "Inspired by XRP fast network  , token Utilit) donation, memes, and gambling.",
+      "type_asset": "cw20",
+      "address": "juno1vwamah7e67v0rdrznxtdjrqw085v09aa7erhdagdavxeh79q60sqt4gkqc",
+      "denom_units": [
+        {
+          "denom": "cw20:juno1vwamah7e67v0rdrznxtdjrqw085v09aa7erhdagdavxeh79q60sqt4gkqc",
+          "exponent": 0
+        },
+        {
+          "denom": "WXRPS",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno1vwamah7e67v0rdrznxtdjrqw085v09aa7erhdagdavxeh79q60sqt4gkqc",
+      "name": "WXRPS Lighting",
+      "display": "WXRPS",
+      "symbol": "WXRPS",
+      "logo_URIs": {
+        "png": "https://images.pexels.com/photos/16466659/pexels-photo-16466659.png"
```

Comment:
png url must point to the Cosmos Chain Registry

Reply:
is there a specific folder i need to upload it to ?>

Reply:
yes, juno/images/